pages/case-studies/yara/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Yara International Design System
+date: 2024-01-19
+layout: LayoutCase
+team: [varya, irina, andriy, alexandra]
+logo: /logo/yara-logo.svg
+description: >
+    Refining Yara design system: in this project, we enreached a React library with 10 new components,
+    upgraded to Storybook@7, improved accessibility, and restructured tokens. Key efforts included
+    infrastructure optimization, thorough documentation overhaul, and customizing the Storybook UI based
+    on user feedback.
```

Comment:
Refining Yara design system: in this project, we enriched a React library with over 10 new components and refactored many outdated ones. We also conducted accessibility audit, devveloped new token  structure and migrated dewsign syste documentation to Storybook@7.
```suggestion
        Refining Yara design system: in this project, we enriched a React library with more than 10 new components and refactored many others. We also conducted an accessibility audit, devveloped new tokens structure and migrated design system documentation to Storybook@7. Key efforts included infrastructure optimization, thorough documentation overhaul, and customizing the Storybook UI based on user feedback.
```